components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbDataFormat.java

```diff
@@ -64,6 +64,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.sun.xml.bind.marshaller.CharacterEscapeHandler;
```

Comment:
Sure. Sorry, wasn't aware of the dependency constraint. I've updated the PR and made it generic to inject any properties for a given JAXB-RI